source/renderer/app/api/utils/localStorageBridge.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type { StorageKey } from '../../../../common/types/electron-store.types';
+
+export const localStorageBridge = async () => {
```

Comment:
This is required to solve the following error when running storybook since `ipcRenderer` is not provided.

```
VM1721 main.c7d2b929e7e498015887.bundle.js:173935 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'on')
    at VM1721 main.c7d2b929e7e498015887.bundle.js:173935
    at new Promise (<anonymous>)
    at _callee$ (VM1721 main.c7d2b929e7e498015887.bundle.js:173918)
    at c (VM1720 vendor.dll.js:31)
    at Generator._invoke (VM1720 vendor.dll.js:31)
    at Generator.next (VM1720 vendor.dll.js:31)
    at asyncGeneratorStep (VM1721 main.c7d2b929e7e498015887.bundle.js:1206)
    at _next (VM1721 main.c7d2b929e7e498015887.bundle.js:1228)
    at VM1721 main.c7d2b929e7e498015887.bundle.js:1235
    at new Promise (<anonymous>)
```